src/wmnds/assets/sass/_vars.scss

```diff
@@ -18,6 +18,7 @@ $palettes: (
   information: #84329b,
   disable: #676869,
   plannedDisruption: #ffdd00,
+  border: #979797,
```

Comment:
Will this color be reused?
If so, you can let it here, if not, move it to the component own scss file.

Reply:
There is an accordion header in the zeplin doc (listed with the tables) that uses it but I'm not sure when that will be needed. I'll just put it in the component scss for now as you suggested.